admin/plugins.php

```diff
@@ -390,3 +458,87 @@ function perflab_render_plugin_card( array $plugin_data ) {
 	</div>
 	<?php
 }
+
+// WordPress AJAX action to handle the button click event.
+add_action( 'wp_ajax_perflab_install_activate_standalone_plugins', 'perflab_install_activate_standalone_plugins_callback' );
+add_action( 'wp_ajax_nopriv_perflab_install_activate_standalone_plugins', 'perflab_install_activate_standalone_plugins_callback' );
+
+/**
+ * Handles the standalone plugin install and activation via AJAX.
+ *
+ * @since n.e.x.t
+ */
+function perflab_install_activate_standalone_plugins_callback() {
+	if ( ! wp_verify_nonce( $_REQUEST['nonce'], 'perflab-install-activate-plugins' ) ) {
+		$status['errorMessage'] = esc_html__( 'Invalid nonce: Please refresh and try again.', 'performance-lab' );
+		wp_send_json_error( $status );
+	}
+
+	if ( ! isset( $_REQUEST['data'] ) ) {
+		$status['errorMessage'] = esc_html__( 'Invalid plugin data: Please refresh and try again.', 'performance-lab' );
+		wp_send_json_error( $status );
+	}
+
+	if ( ! function_exists( 'plugins_api' ) ) {
+		require_once ABSPATH . 'wp-admin/includes/plugin-install.php';
+	}
+	require_once ABSPATH . 'wp-admin/includes/class-wp-upgrader.php';
+	require_once ABSPATH . 'wp-admin/includes/class-wp-ajax-upgrader-skin.php';
+
+	$plugins_to_install = $_REQUEST['data'];
+	$plugins_to_install = explode( ',', $plugins_to_install );
```

Comment:
Nit-pick: Shouldn't this better be called `$plugins_to_activate`? The goal here is to activate those plugins. Installing those that aren't installed is only a partial step to getting there.